### PR TITLE
Add class work `.qmd` files

### PR DESCRIPTION
**Pull request by juliasilge**

### CODE REVIEW

**Comment by juliasilge** [comment]
In `requirements.txt`, we have:
https://github.com/posit-conf-2024/vetiver/blob/b7db92a3567c6bf04a811595c69c0cf1e41525d9/requirements.txt#L12

but when I read the model from the pin, I see this:

> Trying to unpickle estimator RandomForestRegressor from version 1.5.0 when using version 1.5.1. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:
> https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations

Was that model maybe trained with a different version of scikit-learn than what we have pinned here?

File: class-work/01-intro-python.qmd [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+title: Introduction
+subtitle: Deploy and maintain models with vetiver (Python)
+format: html
+---
+
+## Explore the data 👀
+
+```{python}
+import pandas as pd
+housing = pd.read_parquet('./data/housing.parquet')
```
Comment (juliasilge) [inline class-work/01-intro-python.qmd:11]: These are all mixed up with `./data` and `../data` right now. We need to see what happens with working directory for Quarto in VS Code on Workbench and make them correct.

File: class-work/04-monitor-python.qmd [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+```{python}
+## make a visualization
+```
+
+## Create a monitoring dashboard or report 📊
+
+To wrap up, create a model monitoring report and/or dashboard and publish it to Connect. You have a few options:
+
+- Open the `04-monitor-report.qmd`, make any changes you like (for example, different metrics or visualizations), render it, and publish to Connect.
+- Open a fresh copy of the dashboard template with `vetiver.templates.monitoring_dashboard()`, update it to make a dashboard for _your_ model, and publish to Connect.
```
Comment (juliasilge) [inline class-work/04-monitor-python.qmd:79]: So exciting that we have this now! 🎉

**Comment by isabelizimm** [comment]
> Was that model maybe trained with a different version of scikit-learn than what we have pinned here?

Ah, I believe it was. I just updated it so we shouldn't see this warning anymore!

File: class-work/01-intro-python.qmd (hunk 1 above)
Comment (isabelizimm) [reply class-work/01-intro-python.qmd]: fwiw: I tried this out in VSCode Codespaces, which should be pretty identical to Workbench, and `../` was the way to go

**Review by isabelizimm — APPROVED** [review]
looks good! made a few small changes to the python side

**Pull request merged**